content/src/denylist/NoopDenylist.ts

```diff
@@ -0,0 +1,40 @@
+import { DenylistRepository } from '../repository/extensions/DenylistRepository'
+import {
+  Denylist,
+  DenylistMetadata,
+  DenylistSignatureValidationResult,
+  DenylistSignatureValidationStatus
+} from './Denylist'
+import { DenylistTarget, DenylistTargetId, DenylistTargetType } from './DenylistTarget'
+
+export class NoopDenylist extends Denylist {
+  async addTarget(target: DenylistTarget, metadata: DenylistMetadata): Promise<DenylistSignatureValidationResult> {
+    return { status: DenylistSignatureValidationStatus.ERROR }
```

Comment:
We probably should rename this, since this is not a signature validation anymore. Also, would it make sense to add a message explaining why we are failing?

Reply:
Does it make sense to throw an exception with the error message and then the controller will handle the response?

Reply:
I think the current approach makes sense, so we can differentiate between some unwanted error, and a validation error

Reply:
Done!